test/runtests.jl

```diff
@@ -128,9 +128,9 @@ end
     run_all_tests("blas32", reverse(ReferenceBLAS32_jll.LIBPATH_list), :LP64, "", tests = [dgemm, sdot])
 end
 
-@testset "Vanilla ReferenceBLAS_jll (ILP64)" begin
-    run_all_tests("blas", reverse(ReferenceBLAS_jll.LIBPATH_list), :ILP64, "", tests = [dgemm, sdot])
```

Comment:
What is the error that you were seeing here?  Looking through the CI logs, I don't understand.